packages/core/src/lib/components/T/types.ts

```diff
@@ -129,7 +139,8 @@ export type Props<Type extends any> = AnyProps &
   BaseProps<Type> &
   ClassProps<Type> &
   CameraProps<Type> &
-  InstanceProps<Type>
+  InstanceProps<Type> |
```

Comment:
I'm not entirely sure if this is the correct solution... `|`ing seems logical thought since it will overwrite `color` rather than `&` merging it with the broken type definition that doesn't include `string`.